assets/code_example/docs/plugins/resources/maven/updatecli.d/default.yaml

```
@@ -14,8 +14,9 @@ sources:
   default:
     kind: maven
     spec:
-      url: "repo.jenkins-ci.org"
-      repository: "releases"
+      # optionally provide credentials as user:pass@hostname,
```

```suggestion
      # optionally provide credentials as user:pass@hostname
```
suggestion(optional): the comma could induce in error and the comment stays coherent without it.